### ISSUE TITLE
Support react 15.0.0

### ISSUE THREAD

**Issue by saiichihashimoto** [reactjs/express-react-views#74]
React just [updated to version 15.0.0](https://facebook.github.io/react/blog/2016/04/07/react-v15.html). This repository is still using [react 0.14.0](https://github.com/reactjs/express-react-views/blob/8c1ce1fe5264327a72ad47c47dd3848ce6ebe1ac/package.json#L21). When running `npm install --save react@^15.0.0 react-dom@^15.0.0` an error occurs.

```
npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/Cellar/node/4.1.2/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.1.2
npm ERR! npm  v2.14.6
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react@15.0.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer express-react-views@0.10.0 wants react@^0.14.0
```

**Comment by randomailer** [comment]
https://github.com/reactjs/express-react-views/pull/75

**Comment by saiichihashimoto** [comment]
Is there anything stopping this from being published?

**Comment by zpao** [comment]
Published 0.10.1. I was going to try to go through and maybe pick something else up to go in with it but other things popped up.
